Client.Linq/Internal/QueryAggregator.cs

```diff
@@ -83,6 +85,16 @@ internal void AddBucket(string bucket)
             _bucketAssignment = bucket;
         }
 
+        internal void AddImport(string import)
+        {
+            if (_imports == null)
+            {
+                _imports = new HashSet<string>();
+            }
```

Comment:
Please initialise `_imports` in constructor.

Reply:
But then we create an Hashset, even if it is not used. So this would be bad for memory

Reply:
For the sake of consistency with the rest of the code in `QueryAggregator`, such as `AddFilterByTags`, `AddOrder`, etc., it would be better to maintain this format. This will help us in future maintenance. Additionally, feel free to optimize other parts of `QueryAggregator` to align with your code.

Thanks again for your PR 👍